src/main.js

```diff
@@ -0,0 +1,114 @@
+const myLibrary = [];
+localStorage.setItem('items', JSON.stringify(myLibrary));
```

Comment:
Every time the page is loaded the localStorage `items` field is reset to an empty array 🤔.